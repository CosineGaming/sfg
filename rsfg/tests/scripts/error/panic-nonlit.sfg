fn main()
	var x = 5
	panic(x, 5)

