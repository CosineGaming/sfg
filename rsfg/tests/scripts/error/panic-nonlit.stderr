[ERROR] literal value required for panic at 3:8